src/app/utils.ts

```diff
@@ -80,6 +80,25 @@ export default class Utils {
     return totalSec < 0 || totalSec >= 86400;
   }
 
+  public static getColorCode(readKey, cnt) {
```

Comment:
as we have it in Utils, can we remove the copy for readings graph and use this itself.